spikeinterface/core/old_api_utils.py

```diff
@@ -102,35 +140,56 @@ def get_traces(self, start_frame, end_frame, channel_indices):
         return self._oldapi_recording_extractor.get_traces(channel_ids=channel_ids,
                                                            start_frame=start_frame,
                                                            end_frame=end_frame).T
-        
-def create_recording_from_old_extractor(oldapi_recording_extractor)->OldToNewRecording:
+
+
+def create_recording_from_old_extractor(oldapi_recording_extractor) -> OldToNewRecording:
     new_recording = OldToNewRecording(oldapi_recording_extractor)
     return new_recording
 
+
 class OldToNewSorting(BaseSorting):
     """Wrapper class to convert old SortingExtractor to a
     new Sorting in spikeinterface > v0.90
-    
+
     Parameters
     ----------
     oldapi_sorting_extractor : se.SortingExtractor
         sorting extractor from spikeinterface < v0.90
     """
-    
+
     def __init__(self, oldapi_sorting_extractor):
         BaseSorting.__init__(self, sampling_frequency=oldapi_sorting_extractor.get_sampling_frequency(),
                              unit_ids=oldapi_sorting_extractor.get_unit_ids())
-               
+
         sorting_segment = OldToNewSortingSegment(oldapi_sorting_extractor)
         self.add_sorting_segment(sorting_segment)
-        
+
         self.is_dumpable = False
+
+        # add old properties
```

Comment:
This looks like a sad copy/paste.
You should make a function for the 2 situtations.